### DISCUSSION TITLE
iCloud 同步 Obsidian Git Vault 导致 iOS App 无限重启问题

### DISCUSSION THREAD

**Discussion by giscus** [edony-ink/comments#30] [General]
# iCloud 同步 Obsidian Git Vault 导致 iOS App 无限重启问题

折腾记：最近碰到了一个 Obsidian iOS App 无限重启的问题，社区搜索了一下好像没有人报类似的问题，为了能够定位和解决问题学习了一些基本技术原理包括 iCloud 同步原理、Obsidian App 启动原理、obsidian-git 插件移动端实现、File Over App 理念等，接着介绍一下我定位分析问题的思路以及最终的解决办法。

https://www.edony.ink/obsidian-ios-app-infinite-restart-issue/

**Comment by yinan-c** [comment]
我的解决方案是在移动端使用 working copy 同步，目前用下来没有什么问题
https://meganesulli.com/blog/sync-obsidian-vault-iphone-ipad/

**Reply by edonyzpc** [reply]
working copy 是一个插件吗？好像没有找到

**Reply by yinan-c** [reply]
是一个 ios 应用，可以把手机上的本地文件和一个 github repo 同步，这样电脑手机两边都是本地文件 + git 同步，不需要经过 icloud
